cost-analyzer/values.yaml

```diff
@@ -289,6 +289,7 @@ oidc:
   authURL: "https://my.auth.server/authorize" # endpoint for login to auth server
   loginRedirectURL: "http://my.kubecost.url/model/oidc/authorize" # Kubecost url configured in provider for redirect after authentication
   discoveryURL: "https://my.auth.server/.well-known/openid-configuration" # url for OIDC endpoint discovery
+  skipOnlineTokenValidation: false # allows oidc to validate the token offline
```

Comment:
Could we have a clearer comment here?

Something like: "if true, will skip accessing OIDC introspection endpoint for online token verification, and instead try to locally validate JWT claims`.